### DISCUSSION TITLE
Throttling API/Backend on free plan?

### DISCUSSION THREAD

**Discussion by saendu** [supabase/supabase#2477] [Questions]
I am currently developing an app that basically works as a search for a large number of recipes. 

I witness from time to time that my backend is not reachable anymore. My request just fall into a timeout. No error, no 429 status or anything. Is there a throttling logic in place that blocks clients of they do too many requests? Is this related because I am using a free plan? 

I also checked https://status.supabase.io/ no issues were reported. 

*** UPDATE ***
On the supabase dashboard I see an error (not very expressive, I guess it is a timeout): Error connecting to API.

**Comment by vxncetxn** [comment]
Facing the same issue as per your update (Error connecting to API). I think their service is down at the moment? https://status.supabase.io/ says "Service Under Maintenance".

**Reply by saendu** [reply]
Ah maybe it is because of their release week...
